javascript/javascript1/week2/Event-application.js

```diff
@@ -0,0 +1,13 @@
+const Week = [ "Sun","Mon","Tus","Wed","Thu","Fri","Sat"]
+
+function getEventDay(daysAfter){
+    // get now date in milliseconds
+    let todayInMilliseconds = new Date().getTime();
+    // get number of days in milliseconds
+    let daysAfterInMilliseconds = daysAfter * 24 * 60 * 60 * 1000;
```

Comment:
Smart approach. 
But sometimes might not work correctly:
daylight saving time, leap seconds, leap years,  historical periods when timezone is changed
Bit of scary details: https://levelup.gitconnected.com/why-is-programming-with-dates-so-hard-7477b4aeff4c